util/cosmos/iscn.js

```diff
@@ -1,36 +1,29 @@
+import { LcdClient } from '@cosmjs/launchpad';
 import { DEFAULT_ISCN_GAS_PRICE, sendTx } from '../CosmosHelper';
 import { ISCN_PUBLISHERS, ISCN_LICENSES } from './iscnConstant';
-import {
-  ISCN_TESTNET_CHAIN_ID,
-} from '@/constant';
 import { timeout } from '@/util/misc';
 import { apiPostISCNMessageForSign } from '../api/api';
 import { assertOk, queryTxInclusion } from './misc';
 
 const ISCN_DEV_RESTFUL_API = '/api/cosmos/iscn-dev/lcd';
+const ISCN_API_ENDPOINT = 'https://node.iscn-dev.like.co/';
 
 let iscnApi;
-let Cosmos;
-
 async function initISCNCosmos() {
   if (iscnApi) return;
-  ([Cosmos] = await Promise.all([
-    import(/* webpackChunkName: "web3" */ '@lunie/cosmos-api'),
-  ]));
-  if (Cosmos.default) Cosmos = Cosmos.default;
-  iscnApi = new Cosmos(ISCN_DEV_RESTFUL_API, ISCN_TESTNET_CHAIN_ID);
+  iscnApi = LcdClient.withExtensions({ ISCN_API_ENDPOINT });
```

Comment:
Similarly, should use `ISCN_DEV_RESTFUL_API`, if an absolute path is needed, should concat `EXTERNAL_URL` and `ISCN_DEV_RESTFUL_API`